audio/audacity/files/patch-src-memoryx-disable-snowleopard-fix.diff

```diff
@@ -0,0 +1,11 @@
+--- a/src/MemoryX.h.old	2017-11-18 15:43:36.000000000 -0800
++++ b/src/MemoryX.h	2017-11-18 15:43:51.000000000 -0800
+@@ -12,7 +12,7 @@
+ // std:: containers knowing about rvalue references
+ #undef __AUDACITY_OLD_STD__
+ 
+-#if defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED <= __MAC_10_6
```

Comment:
Why is this required? Are you expecting that someone might try to build against libstdc++ on 10.7+ (other than you during testing)?

(This does remind me I should pick up my work on a `-stdlib=libc++` option for GCC again!)